### DISCUSSION TITLE
Thanks, Again!

### DISCUSSION THREAD

**Discussion by proteus-dev** [DeabLabs/cannoli#23] [General]
I just wanted to thank you one more time for creating Cannoli.

I just created a Cannoli-Centered-Vault that we started using productively in a small team for creating educational and marketing-related texts. 

It took me only one day to set everything up and get my colleagues up and running in the system. I mention this because this is the first time for them to:
- use Obsidian
- use Cannoli
- use an AI-based service

The result:
They understand everything they needed to know and can now indenpendently use everything. They are blown away by the ease-of-use, naturally of both, Cannoli and AI-based services, and start thinking about creating their own Cannolis. 

Thanks again, I cannot say it often enough! :)

**Comment by blindmansion** [comment]
Thank you! That is exactly why we made cannoli, and its super cool to know it's being used for stuff like this.
